robotpy/src/main/java/io/github/robotpy/plugins/robotpy/feature/feature.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file doing here? It already exists elsewhere.

Reply:
I had some issues where there were files needed there that were actually in the robotpy.features folder. Copying them over fixed that issue.